before-install/multiple-instances.md

```diff
@@ -2,27 +2,28 @@
 
 If you need multiple WordPress instances, there are three types of installations based on system architecture, or a combination of WordPress instances and databases:
 
-1. WordPress Multisite feature: a single WordPress instance (with multiple sites created within the same WP instance) sharing a single database instance.
+1. WordPress Multisite Network: a single WordPress instance (with multiple sites created within the same WP instance) sharing a single database instance.
 2. Single Database: multiple WordPress instances sharing a single database instance.
 3. Multiple Databases: multiple WordPress instances with each instance using its own databases instance.
 
-![](https://wordpress.org/documentation/files/2022/06/multisite_db_layout-1024x469.jpg)
+![](https://wordpress.org/support/files/2022/06/multisite_db_layout-1024x469.jpg)
```

Comment:
We should use “documentation” files:
https://wordpress.org/documentation/files/2022/06/multisite_db_layout-1024x469.jpg